File: gex/src/model/BarMatchPlayerLeft.ts

export class BarMatchPlayerLeft {
    public gameID: string = "";
    public playerID: number = 0;
    public reason: number = 0;
    public gameTime: number = 0;

    public static parse(elem: any): BarMatchPlayerLeft {
        return {
            ...elem
        };
    }

}

File: gex/Models/Db/BarMatchPlayerLeft.cs
using Dapper.ColumnMapper;
using gex.Code;

namespace gex.Models.Db {

    [DapperColumnsMapped]
    public class BarMatchPlayerLeft {

        [ColumnMapping("game_id")]
        public string GameID { get; set; } = "";
 
        [ColumnMapping("player_id")]
        public byte PlayerID { get; set; }

        [ColumnMapping("reason")]
        public byte Reason { get; set; }

        [ColumnMapping("game_time")]
        public float GameTime { get; set; }

        [ColumnMapping("index")]
        public int Index { get; set; }

    }
}
